src/net.h

```diff
@@ -762,6 +762,12 @@ class CNode
         vInventoryToSend.push_back(inv);
     }
 
```

Comment:
please add doxygen comment (and don't hold back if you want to also doc a few of the functions near your addition!)

Reply:
ok will do

Reply:
done